src/components/classifier/FieldGuide.js

```diff
@@ -1,305 +1,316 @@
-import React, { Component } from 'react'
+import React, {Component} from 'react'
 import {
-  Animated,
-  Dimensions,
-  Easing,
-  PanResponder,
-  Platform,
-  ScrollView,
-  TouchableOpacity,
-  View
+    Animated,
+    Dimensions,
+    Easing,
+    PanResponder,
+    Platform,
+    ScrollView,
+    TouchableOpacity,
+    View
 } from 'react-native'
 import EStyleSheet from 'react-native-extended-stylesheet'
 import FieldGuideItemDetail from './FieldGuideItemDetail'
 import FieldGuideItemRow from './FieldGuideItemRow'
 import Icon from 'react-native-vector-icons/FontAwesome'
 import PropTypes from 'prop-types';
-import { addIndex, isEmpty, map } from 'ramda'
+import {addIndex, isEmpty, map} from 'ramda'
+
+import * as colorModes from '../../actions/colorModes'
 
 const MAX_DEFAULT_HEIGHT = Dimensions.get('window').height * .6
 const MIN_HEIGHT = 33
 const MAX_DRAG_HEIGHT = Dimensions.get('window').height - (Platform.OS === 'ios' ? 100 : 110)
 
 export class FieldGuide extends Component {
-  constructor(props) {
-    super(props)
+    constructor(props) {
+        super(props)
 
-    this.state = {
-      selectedItem: {},
-      heightAnim: new Animated.Value(0),
-      height: 0,
-      headerHeight: 0
+        this.state = {
+            selectedItem: {},
+            heightAnim: new Animated.Value(0),
+            height: 0,
+            headerHeight: 0
+        }
     }
-  }
 
-  componentWillMount() {
-     this._panResponder = PanResponder.create({
-      onStartShouldSetPanResponder: () => true,
-      onStartShouldSetPanResponderCapture: () => true,
-      onMoveShouldSetPanResponder: () => true,
-      onMoveShouldSetPanResponderCapture: () => true,
+    componentWillMount() {
```

Comment:
Good call. I ended up updating this and one other warning along the same lines. 

One interesting thing I discovered while reading the documentation on these warnings is that _most_ users _don't_ want to use the suggested rename, but instead want to go with `componentDidMount()` here.

Figured I'd mention that in this comment in case anyone comes across this PR and is about to have to do something similar :). I wonder why react has chosen to recommend the deprecation namespace rename rather than switching to the new API.